packages/compass-collection/src/components/collection-header/collection-header.tsx

```diff
@@ -9,65 +16,90 @@ import CollectionHeaderActions from '../collection-header-actions';
 import ReadOnlyBadge from './read-only-badge';
 import TimeSeriesBadge from './time-series-badge';
 import ViewBadge from './view-badge';
+import CollectionStats from '../collection-stats';
+import type { CollectionStatsObject } from '../../modules/stats';
 
 const collectionHeaderStyles = css({
-  paddingTop: '15px',
-  paddingBottom: '5px',
-  minHeight: '64px',
+  paddingTop: spacing[3],
+  paddingBottom: spacing[1],
+  minHeight: spacing[6] + spacing[1],
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'space-between',
   background: uiColors.white,
```

Comment:
Not totally related, looks like this background might not be needed anymore? If we remove this background setting it looks like the background adjusts based on the background color. (with the env variable `COMPASS_LG_DARKMODE`)
```suggestion
```

<img width="898" alt="Screen Shot 2022-04-20 at 5 59 13 PM" src="https://user-images.githubusercontent.com/1791149/164332542-9fd11838-ba3d-48a0-93e2-a00f5845a33c.png">
<img width="878" alt="Screen Shot 2022-04-20 at 6 03 11 PM" src="https://user-images.githubusercontent.com/1791149/164332544-9482ae42-1367-47a3-b47c-3fadcb6a8a91.png">